src/services/venue-service.ts

```diff
@@ -34,14 +34,16 @@ export const DEFAULT_VENUE: Venue = {
 export class VenuesService {
```

Comment:
I noticed that the class name is `VenuesService` (plural) vs. the file name which is `venue-service.ts` (singular). Should the convention be that file name matches class name? Also since `venue` is our entity would it be better for the naming to singular?